modules/mapper-extras/src/main/java/org/elasticsearch/index/mapper/extras/RankFeaturesFieldMapper.java

```diff
@@ -77,7 +80,7 @@ public static final class RankFeaturesFieldType extends MappedFieldType {
         private final boolean positiveScoreImpact;
 
         public RankFeaturesFieldType(String name, Map<String, String> meta, boolean positiveScoreImpact) {
-            super(name, false, false, false, TextSearchInfo.NONE, meta);
+            super(name, true, false, false, TextSearchInfo.SIMPLE_MATCH_ONLY, meta);
```

Comment:
@jpountz for the mapping consistency tests to pass, I had to indicate that since this field allows `term` queries, is that its `indexed` and that its `SIMPLE_MATCH_ONLY`. This seems OK to me as "technically" it was indexed as a term & frequency, just didn't know if this broke other things.

Reply:
It looks ok to me. Maybe @romseygeek can confirm?